internal/jimm/access.go

```diff
@@ -717,18 +717,27 @@ func (j *JIMM) CountGroups(ctx context.Context, user *openfga.User) (int, error)
 	return count, nil
 }
 
-// GetGroup returns a group based on the provided UUID.
-func (j *JIMM) GetGroupByID(ctx context.Context, user *openfga.User, uuid string) (*dbmodel.GroupEntry, error) {
-	const op = errors.Op("jimm.AddGroup")
+// getGroup returns a group based on the provided UUID or name.
+func (j *JIMM) getGroup(ctx context.Context, user *openfga.User, group *dbmodel.GroupEntry) (*dbmodel.GroupEntry, error) {
+	const op = errors.Op("jimm.getGroup")
```

Comment:
`
```suggestion
        if group != nil && group.name != "" {
	   const op = errors.Op("jimm.GetGroupByName")
	} else {
	...
	}
```
otherwise it will be very hard to tell from the logs which API is failing.

Reply:
In this case I'd rather duplicate the code and keep them separate. But I don't think it'd be difficult to point to the failing one as the caller of this has errors.Op which include the name of the actual endpoint that's being requested.
e.g. `access_control.go:64`